tasks/main.yml

```diff
@@ -172,6 +183,12 @@
     mode: 0664
   when: (devimg_dir is defined) and (devimg_dir | length > 0)
 
+- name:  Check Quamotion XCUITRunner
```

Comment:
I'm not entirely convinced this is the right way. For example, if no device is connected, the connected device is not trusted, or multiple devices are connected, this check will fail.